src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -1132,6 +1136,7 @@ CREATE TABLE transcription_document
 ,file_name                   CHARACTER VARYING             NOT NULL
 ,file_type                   CHARACTER VARYING             NOT NULL
 ,file_size                   INTEGER                       NOT NULL
+,checksum                    CHARACTER VARYING             NOT NULL

```

Comment:
Similar to my comment on the v53 PR media checksum...
Why do we need the checksum column on `transcription_document` table as well as `external_object_directory`?

Is this expected to be calculated to correspond to the file hash value in the CONTENT-MD5 tag as a base64 encoded representation of the binary MD5 hash value

e.g. openssl md5 -binary "Test Document.doc" | base64
KQ9vVogyRdsnEvxyNQz77g==

FYI @hemantasharma1129

Reply:
All checksum fields are moved to respective tables i.e media, transcription_document, annotation_document. Yes the checksum will be the MD5 hash value. I agree the checksum field in external_object_directory should be removed.